pkg/doc/jose/encrypter_test.go

```diff
@@ -0,0 +1,109 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package jose
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/google/tink/go/keyset"
+	"github.com/square/go-jose/v3"
+	"github.com/square/go-jose/v3/json"
+	"github.com/stretchr/testify/require"
+
+	"github.com/hyperledger/aries-framework-go/pkg/crypto/tinkcrypto/primitive/composite/ecdhes"
+	ecdhessubtle "github.com/hyperledger/aries-framework-go/pkg/crypto/tinkcrypto/primitive/composite/ecdhes/subtle"
+)
+
+func TestJWEEncrypt(t *testing.T) {
+	_, err := NewJWEEncrypt(nil)
+	require.Error(t, err, "NewJWEEncrypt should fail with empty recipientPubKeys")
+
+	recECKeys, _ := createRecipients(t, 20)
+
+	jweEncrypter, err := NewJWEEncrypt(recECKeys)
+	require.NoError(t, err, "NewJWEEncrypt should not fail with non empty recipientPubKeys")
+
+	jwe, err := jweEncrypter.Encrypt([]byte("some msg"), []byte("aad value"))
+	require.NoError(t, err)
+	require.Equal(t, len(recECKeys), len(jwe.Recipients))
+
+	serializedJWE, err := jwe.Serialize(json.Marshal) // must use Go Jose's json.Marshal fct (not encoding.json.Marshal)
```

Comment:
Is fct short for function? I think "func" (if it fits within the character limit) or "fn" is a more common abbreviation

Reply:
moved comment line above code..